### PR TITLE
new: Add Lab 2.16 release notes and connection types docs

### PR DESCRIPTION
**Pull request by tonilastre**
### Description

Add docs for Lab 2.16 including updates to the release notes page and an additional page for connection types.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [x] New documentation page, release related

### Related PRs and issues

-

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/data-visualization/install-and-connect.mdx [hunk 1]
```diff
@@ -83,18 +83,24 @@ If you encounter a security warning while installing Memgraph Lab on Windows, yo
...
-- Leave the `Username` and `Password` fields **empty**.
-- The `Host` field can be either **`localhost`**, **`127.0.0.1`** or
-   **`0.0.0.0`**, or change it appropriately.
-- The `Port` field should be **`7687`**. Every Memgraph instance is listening
-   on this port by default.
-- The `Encrypted` option should be **disabled** and display `SSL Off` by
-   default.
+Now, input the connection information:
+
+- **Host**: Enter `localhost`, `127.0.0.1`, or `0.0.0.0`, or
```
Comment (katarinasupe) [inline pages/data-visualization/install-and-connect.mdx:95]: Do we want to mention host.docker.internal here or not?

File: pages/data-visualization/user-manual/connection-types.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import { Callout } from 'nextra/components'
+
+# Connection types
+
+Starting from Lab 2.16, Memgraph Lab supports three types of
+connections to your Memgraph instances:
+
+1. Connect to a standalone instance
+2. Connect to a main or replica instance
+3. Connect to a HA cluster with automatic failover
```
Comment (katarinasupe) [inline pages/data-visualization/user-manual/connection-types.mdx:15]: I would add links to sections (anchors) here

**Review by katarinasupe — APPROVED** [review]
Overall, I think the docs are good, it's all clear

File: pages/data-visualization/install-and-connect.mdx (hunk 1 above)
Comment (tonilastre) [reply pages/data-visualization/install-and-connect.mdx:95]: I think it makes sense to mention it. Will do.

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

File: pages/data-visualization/user-manual/connection-types.mdx (hunk 2 above)
Comment (katarinasupe) [reply pages/data-visualization/user-manual/connection-types.mdx]: @tonilastre I though of linking to below sections, not to different docs pages

File: pages/data-visualization/install-and-connect.mdx [hunk 3]
```diff
@@ -83,18 +83,24 @@ If you encounter a security warning while installing Memgraph Lab on Windows, yo
...
-   **`0.0.0.0`**, or change it appropriately.
-- The `Port` field should be **`7687`**. Every Memgraph instance is listening
-   on this port by default.
-- The `Encrypted` option should be **disabled** and display `SSL Off` by
-   default.
+Now, input the connection information:
+
+- **Host**: Enter `localhost`, `host.docker.internal`, `127.0.0.1`,
+  `0.0.0.0`, or modify it as needed.
+- **Port**: Default is `7687`, which Memgraph uses by default.
```
Comment (hal-eisen-MG) [inline pages/data-visualization/install-and-connect.mdx:97]: Redundant word "default"

File: pages/data-visualization/install-and-connect.mdx [hunk 4]
```diff
@@ -83,18 +83,24 @@ If you encounter a security warning while installing Memgraph Lab on Windows, yo
...
+- **Database name**: Leave empty unless connecting to a
+  [multi-tenant Memgraph instance](/database-management/multi-tenancy).
+- **Logs port**: Default is `7444`, the port for the web socket
+  logs.
+- **Encrypted**: Ensure this option is disabled by default.
```
Comment (hal-eisen-MG) [inline pages/data-visualization/install-and-connect.mdx:102]: What value will enable or disable? Is that "true", "false", "0", "1", or something else?

File: pages/data-visualization/install-and-connect.mdx (hunk 3 above)
Comment (tonilastre) [reply pages/data-visualization/install-and-connect.mdx:97]: Hmm but is it? Maybe I should restructure this, but the Lab default is 7687, which Memgraph uses by default. It is two different defaults (Lab UI vs Memgraph)

Comment (tonilastre) [inline pages/data-visualization/install-and-connect.mdx:97]:
```suggestion
- **Port**: Enter `7687`, which Memgraph uses by default.
```

File: pages/data-visualization/user-manual/connection-types.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+![](/pages/data-visualization/lab-user-manual/connection-types/conn-type-standalone-input.png)
+
+This is the regular Bolt connection to a single Memgraph
+instance. You will need to define:
+
+- **Host**: Either `localhost`, `127.0.0.1`, `0.0.0.0`,
+  or another appropriate address.
+- **Port**: Default is `7687`, which Memgraph uses by
+  default.
```
Comment (tonilastre) [inline pages/data-visualization/user-manual/connection-types.mdx:29]:
```suggestion
- **Port**: Enter `7687`, which Memgraph uses by default.
```

File: pages/data-visualization/install-and-connect.mdx (hunk 4 above)
Comment (tonilastre) [reply pages/data-visualization/install-and-connect.mdx:102]: Regarding the value, it is in the second sentence: "Enable it if your Memgraph has SSL enabled."

**Pull request merged**